src/screens/OtherProfile/index.js

```diff
@@ -283,7 +204,7 @@ const OtherProfile = () => {
     };
     const result = await setFollow(data);
     if (result.code === 200) {
-      fetchOtherProfile();
+      refetchOtherProfile();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same comment as for line 188.



<!-- This is an auto-generated comment by OSS CodeRabbit -->